cmd/aigw/docker-compose.yaml

```diff
@@ -99,3 +114,24 @@ services:
           -d "{\"model\":\"$$EMBEDDINGS_MODEL\",\"input\":\"How do I reset my password?\"}"
     extra_hosts:  # localhost:host-gateway trick doesn't work with aigw
       - "host.docker.internal:host-gateway"
+
+  # mcp is a test client for calling MCP tools through aigw.
+  mcp:
+    image: ghcr.io/modelcontextprotocol/inspector:latest
+    container_name: mcp
+    profiles: ["test"]
+    entrypoint: node
+    command:
+      - cli/build/cli.js
+      - --cli
+      - http://aigw:1975/mcp
+      - --method
+      - tools/call
+      - --tool-name
+      - kiwi__search-flight
+      - --tool-arg
+      - flyFrom=NYC
+      - --tool-arg
+      - flyTo=LAX
+      - --tool-arg
+      - departureDate=15/12/2025
```

Comment:
[nit] the Kiwi flight search tool fails when dates in the past are passed to it (and probably a date far away in the future will produce no results). Should we use a different MCP server, like the `dreamtap` you used before, where we know there will always be an answer? (same comment applies to the otel docker-compose and tests that are actually using it)

Reply:
We can use a calendar to pick next week meanwhile